.readthedocs.yml

```diff
@@ -13,6 +13,6 @@ sphinx:
 
 python:
   install:
-    - requirements: requirements.txt
+    - requirements: docs-requirements.txt
```

Comment:
Can `starknet.py` be installed with all the optional dependencies from pypi?

Reply:
Is there any point of installing starknet.py from pypi that way? I

Reply:
I don't know, just curious though. 😄